### ISSUE TITLE
add dns entry for upload instead of cloudfront

### ISSUE THREAD

**Issue by kookster** [PRX/www.prx.org#244]

**Comment by kookster** [comment]
@farski, I think you made a dns entry for upload, yes?

**Comment by kookster** [comment]
no, wait, this is for the upload target on s3/cloudfront - this needs its own entry

**Comment by farski** [comment]
I added: `userupload.prxu.org. CNAME dsn15m3yob5tf.cloudfront.net`; has not been tested yet.

**Comment by farski** [comment]
Though this won't be an option to deploy until we have a cert for prxu.org.
